### PR TITLE
Stream validations

### PR DESCRIPTION
**Pull request by vanruch**
# Checklist:

- [x] Code follows the style guidelines
- [x] Documentation updated
- [x] Corner cases evaluated and handled
- [x] Automatic tests added/update
- [x] Automatic tests pass
- [x] Software runs (locally or hosted)
- [x] Sanity check passed
- [x] Self-review passed

### CODE REVIEW

File: src/services/data_model_engine.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ import {NotFoundError, PermissionError, ValidationError} from '../errors/errors'
 import {getTimestamp} from '../utils/time_utils';
 import {pick, put} from '../utils/dict_utils';
 import allPermissions from '../utils/all_permissions';
+import {PassThrough} from 'stream';
```
Comment (krzysztof-jelski) [inline src/services/data_model_engine.js]: Unused import.

File: test/services/data_model_engine.js [hunk 2]
```diff
@@ -1248,7 +1267,11 @@ describe('Data Model Engine', () => {
         bundleRepository: mockBundleRepository
       });
 
-      await modelEngine.uploadAcceptedBundleCandidates({success: async () => {}, fail: async () => {}});
```
Comment (krzysztof-jelski) [inline test/services/data_model_engine.js]: I prefer the previous formatting

File: config/config.js [hunk 3]
```diff
@@ -49,5 +49,7 @@ module.exports = Object.freeze({
   workerLogsTTLInSeconds: process.env.WORKER_LOGS_TTL || 172800, // 2 days
   workerTasksTTLInSeconds: process.env.WORKER_TASKS_TTL || 600, // 10 minutes
 
-  migrationSleepTimeInSeconds: process.env.MIGRATION_SLEEP_TIME || 10
+  migrationSleepTimeInSeconds: process.env.MIGRATION_SLEEP_TIME || 10,
+
+  supportDeprecatedBundles: process.env.DISABLE_SUPPORT_FOR_DEPRECATED_BUDLES !== 'true'
```
Comment (Anze1m) [inline config/config.js]: Why not just `supportDeprecatedBundles: process.env.ENABLE_SUPPORT_FOR_DEPRECATED_BUDLES`?
